xarray/tests/test_backends_datatree.py

```diff
@@ -24,6 +24,88 @@
     pass
 
 
+@pytest.fixture(scope="module")
```

Comment:
I added these fixtures to this module since we use these zarr stores and nc files more than once.